app/javascript/app/components/ndc-sdg/ndc-sdg-linkages-content/ndc-sdg-linkages-content-component.jsx

```diff
@@ -17,20 +18,31 @@ class NdcSdgLinkagesContent extends PureComponent {
       goalHover,
       targetHover,
       handleGoalHover,
+      handleGoalTap,
       handleTargetHover
     } = this.props;
     return (
-      <div className={cx(layout.content, styles.grid)}>
-        <NdcsSdgsMetaProvider />
-        <NdcSdgLinkagesTable
-          goalHover={goalHover}
-          onGoalHover={handleGoalHover}
-          targetHover={targetHover}
-          onTargetHover={handleTargetHover}
-        />
-        <NdcSdgLinkagesMap goalHover={goalHover} targetHover={targetHover} />
-        <ModalMetadata />
-      </div>
+      <TabletLandscape>
+        {matches => (
+          <div
+            className={cx({ [layout.content]: matches === true }, styles.grid)}
```

Comment:
Maybe a better variable name could help to understand what it does, for example, an `isTablet` ? also we don't need to compare with `===true` and it's already a boolean